src/olympia/amo/management/commands/compress_assets.py

```diff
@@ -198,7 +198,7 @@ def _minify(self, ftype, file_in, file_out):
         """Run the proper minifier on the file."""
         if ftype == 'js' and hasattr(settings, 'UGLIFY_BIN'):
             opts = {'method': 'UglifyJS', 'bin': settings.UGLIFY_BIN}
-            run_command('{uglify} -v -o {target} {source} -m'.format(
+            run_command('{uglify} -o {target} {source} -m'.format(
```

Comment:
You decided not to bother with `--verbose`?

Reply:
I tried it and there wasn't any difference in the output, weirdly. I decided I had already spent too much time on this...